css/classify.styl

```diff
@@ -124,6 +124,10 @@
   & + .workflow-task
     margin-top: 2em
 
+  .markdown:first-child
+    :first-child
```

Comment:
no that was intentional... it looks for the first occurrence of `.markdown`, and then within that for the first child, regardless of whether it's a `p` or `img` or something else